ui/cypress/seeds/model_api_data.js

```diff
@@ -173,7 +171,7 @@ const genBaseModel = (id, overrides={}) => {
     },
     // "next_version": "string", // would mean the current version is deprecated in favor of next version
     // "prev_version": "string",
-    "is_published": true,
+    "is_published": false,
```

Comment:
I had set this up as published by default, which is problematic. That is, seed/mock/test models created were already marked as published by default, which caused problems on model registration tests. `overrides` was taking care of that, but there was no reason to default to published to begin with so it is unset now.